include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -68,23 +94,39 @@ namespace SuperiorMySqlpp
             }
 
         protected:
+            /**
+             * This method implements the store() specialized behavior.
```

Comment:
What does actually mean "specialized behaviour"?

Reply:
Will clarify in more detailed description. FYI, the same method exist for subclass that has *use*() strategy and for subclass that does *store*() strategy - it does nothing for *use*, but calls mysql_stmt_store_result for *store*.